Sprint 6 Challenge/Sprint 6 Challenge/View Controller/ViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ViewController.swift
+//  Sprint 6 Challenge
+//
+//  Created by Spencer Curtis on 8/29/18.
+//  Copyright © 2018 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet weak var lockImageView: UIImageView!
+    @IBOutlet weak var customControl: CustomControl!
+    @IBOutlet weak var resetButton: UIBarButtonItem!
+    
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return .lightContent
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        resetButton.tintColor = .clear
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        navigationController?.navigationBar.barStyle = .black
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        
+    }
+
+
+    @IBAction func updateLock(_ slider: CustomControl) {
+        
+        if !slider.shouldUnlock {
+            
+        } else {
+            lockImageView.image = #imageLiteral(resourceName: "Unlocked")
+            
```

Comment:
Here you need to add a way to lock the customControl so that the thumb slider locks in place until the reset button is pressed. using something like 
```customControl.isEnabled = false``` on line 43 would suffice. Use the app before making this change and then again after. much better yeah?